sdk/eventhub/Azure.Messaging.EventHubs/tests/Compatibility/TrackOneExceptionExtensionsTests.cs

```diff
@@ -16,7 +16,7 @@ namespace Azure.Messaging.EventHubs.Tests
     /// </summary>
     ///
     [TestFixture]
-    [Parallelizable(ParallelScope.Children)]
+    [Parallelizable(ParallelScope.All)]
```

Comment:
Worth adding one of these at the assembly level?

Reply:
I may,  There are sets of tests, like the diagnostics stuff, that can't be run in parallel.  I'm not exactly sure that NUnit respects overrides on individual classes when you give it direction at the assembly level.  When we finish moving the diagnostics stuff over and I can make sure that its safe to just override them with `[NonParallizable]`, I'll revisit.